src/components/KTruncate/KTruncate.vue

```diff
@@ -228,24 +229,14 @@ const widthStyle = computed((): Record<string, string> => {
 })
 
 onMounted(() => {
-  resizeObserver.value = new ResizeObserver(entries => {
-    // Wrapper 'window.requestAnimationFrame' is needed for disabling "ResizeObserver loop limit exceeded" error in DD
-    window.requestAnimationFrame(() => {
-      if (!Array.isArray(entries) || !entries.length) {
-        return
-      }
-      // Actual code
-      setWrapperHeight()
-    })
-  })
+  resizeObserver.value = ResizeObserverHelper.create(setWrapperHeight)
+
   resizeObserver.value.observe(kTruncateContainer.value as HTMLDivElement)
   updateToggleVisibility()
 })
 
 onBeforeUnmount(() => {
-  if (resizeObserver.value) {
-    resizeObserver.value.unobserve(kTruncateContainer.value)
-  }
+  resizeObserver.value?.unobserve(kTruncateContainer.value as HTMLDivElement)
```

Comment:
issue: this should be done in `onUnmounted` instead:

> Use this hook to clean up manually created side effects such as timers, DOM event listeners or server connections.

[Source](https://vuejs.org/api/composition-api-lifecycle.html#onunmounted)